### ISSUE TITLE
Jenkins marian-regression-tests-with-cudnn #100 failed

### ISSUE THREAD

**Issue by tomekd** [marian-nmt/marian-dev#252]
Build 'marian-regression-tests-with-cudnn' is failing!

Last 50 lines of build output:

```
[...truncated 24.65 KB...]
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/en-de/ [6546/6546] -> "wmt17_systems/en-de/index.html.tmp" [1]
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/en-de/?C=N;O=D [6546/6546] -> "wmt17_systems/en-de/index.html?C=N;O=D.tmp" [1]
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/en-de/?C=M;O=A [6546/6546] -> "wmt17_systems/en-de/index.html?C=M;O=A.tmp" [1]
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/en-de/?C=S;O=A [6546/6546] -> "wmt17_systems/en-de/index.html?C=S;O=A.tmp" [1]
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/en-de/?C=D;O=A [6546/6546] -> "wmt17_systems/en-de/index.html?C=D;O=A.tmp" [1]
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/en-de/?C=N;O=A [6546/6546] -> "wmt17_systems/en-de/index.html?C=N;O=A.tmp" [1]
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/en-de/?C=M;O=D [6546/6546] -> "wmt17_systems/en-de/index.html?C=M;O=D.tmp" [1]
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/en-de/?C=S;O=D [6546/6546] -> "wmt17_systems/en-de/index.html?C=S;O=D.tmp" [1]
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/en-de/?C=D;O=D [6546/6546] -> "wmt17_systems/en-de/index.html?C=D;O=D.tmp" [1]
FINISHED --2018-06-11 15:15:46--
Total wall clock time: 0.02s
Downloaded: 9 files, 58K in 0s (512 MB/s)
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/scripts/ [1532/1532] -> "wmt17_systems/scripts/index.html.tmp" [1]
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/scripts/?C=N;O=D [1532/1532] -> "wmt17_systems/scripts/index.html?C=N;O=D.tmp" [1]
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/scripts/?C=M;O=A [1532/1532] -> "wmt17_systems/scripts/index.html?C=M;O=A.tmp" [1]
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/scripts/?C=S;O=A [1532/1532] -> "wmt17_systems/scripts/index.html?C=S;O=A.tmp" [1]
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/scripts/?C=D;O=A [1532/1532] -> "wmt17_systems/scripts/index.html?C=D;O=A.tmp" [1]
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/scripts/?C=N;O=A [1532/1532] -> "wmt17_systems/scripts/index.html?C=N;O=A.tmp" [1]
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/scripts/?C=M;O=D [1532/1532] -> "wmt17_systems/scripts/index.html?C=M;O=D.tmp" [1]
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/scripts/?C=S;O=D [1532/1532] -> "wmt17_systems/scripts/index.html?C=S;O=D.tmp" [1]
2018-06-11 15:15:46 URL:http://data.statmt.org/wmt17_systems/scripts/?C=D;O=D [1532/1532] -> "wmt17_systems/scripts/index.html?C=D;O=D.tmp" [1]
FINISHED --2018-06-11 15:15:46--
Total wall clock time: 0.01s
Downloaded: 9 files, 13K in 0s (88.4 MB/s)
cd models && bash ./download-char-s2s.sh
/var/lib/jenkins/workspace/marian-regression-tests-with-cudnn/models
mkdir -p data
cd data && bash ./download-data.sh
Downloading europarl.de-en/corpus.bpe.de.gz ...
Downloading europarl.de-en/corpus.bpe.en.gz ...
+ CUDA_VISIBLE_DEVICES=1 bash ./run_mrt.sh tests/examples/mnist
[06/11/2018 15:15:47] Using Marian: /var/lib/jenkins/workspace/marian-regression-tests-with-cudnn/tools/marian
[06/11/2018 15:15:47] Using CUDNN: USE_CUDNN:BOOL=ON
[06/11/2018 15:15:47] Using MKL: MKL_ROOT:PATH=/opt/intel/mkl
[06/11/2018 15:15:47] Using number of devices: 1
[06/11/2018 15:15:47] Using CUDA visible devices: 1
[06/11/2018 15:15:47] Checking directory: tests/examples/mnist
[06/11/2018 15:15:47] Running setup script
[06/11/2018 15:15:47] Running tests/examples/mnist/test_mnist_ffnn.sh ... failed
[06/11/2018 15:15:50] Test took 00:00:2.249s
[06/11/2018 15:15:50] Running tests/examples/mnist/test_mnist_lenet.sh ... failed
[06/11/2018 15:15:52] Test took 00:00:1.944s
---------------------
Failed:
  - tests/examples/mnist/test_mnist_ffnn.sh
  - tests/examples/mnist/test_mnist_lenet.sh
---------------------
Ran 2 tests in 00:00:4.259s, 0 passed, 0 skipped, 2 failed
Build step 'Execute shell' marked build as failure

```

Changes since last successful build:
No changes
No changes
No changes
No changes
- [Roman Grundkiewicz] 460969d0b27ecaf9a4ebe07ae17a5a46e165c762 - Reduce requirements

No changes
- [Roman Grundkiewicz] dcce796ac734ba47bd8717fe907cd0a4ef979d81 - Add missing acknowledgments

- [Roman Grundkiewicz] f2afd7809afd9598949a338c9047e9056ff75be5 - Fix tests on GTX 1080

- [Roman Grundkiewicz] 4aa900f8e51d1e3798ce1962ac5b3b2dba3ae793 - Suppress 'bc command not found'

- [Roman Grundkiewicz] 478d569bbcbf907a1e6f2c6577acde40c8efa5bb - Add test for restoring multiple stalled validators

No changes
- [Roman Grundkiewicz] 0c6de5507a7edf0b33937ef32d6b783a6e9022e3 - Update tests

- [Roman Grundkiewicz] 4e0d3451d9f420bed7f1b9d3e137d4ec4705f557 - Fix multi-node tests

- [Roman Grundkiewicz] aa1c1d74e90dc2c8f7dd54f80bc7badd0c93f8e8 - Update tests for Iris/MNIST examples

- [Roman Grundkiewicz] 8c96c72dd98a52661482d90303625e40f311ce1c - Move optimizer tests to separate directory

- [Roman Grundkiewicz] ae4762e4214ea6a8ba21799ce128ee4e84ed5b58 - Do not remove .log files

- [Roman Grundkiewicz] 02003f855b4946919ba14220afce58c12f0910e2 - Rename tests

- [Roman Grundkiewicz] 396367d98120a88f9367ef423392a4f196de97e7 - Add test for restarting finished training

- [Roman Grundkiewicz] 1c08d994f82707edfe1c1ee95d53ebb2e09cc48d - Update tests

No changes
No changes
No changes
No changes
No changes
No changes
- [Marcin Junczys-Dowmunt] 54d44c8dde64f08cb02f67fd6feece9033d8f9c3 - adjust tests/training/restart/test_restarting_stalled_validators.sh

- [Marcin Junczys-Dowmunt] 025feb5b778e20a430cd95a4fed99fa4f21b119c - adjust

- [Marcin Junczys-Dowmunt] d41ebec521e787f89d24efdfd0d851611a35b293 - adjust tests/training/restart/test_restoring_validation.sh

- [Marcin Junczys-Dowmunt] 97816d8597ecb0372a50677bd66a125daea2c488 - adjust tests/training/weights/test_validation.sh

- [Marcin Junczys-Dowmunt] 7509d7b7ab6eb02aa4a466a0e3cc6c3311a311d9 - adjust tests for final evaluation

- [Marcin Junczys-Dowmunt] 7b28da31c97635f50da842409171d4a8bb2a308e - adjust validation set for epochs

- [Marcin Junczys-Dowmunt] c86ccb046902512a6704f7359b25de3fc821287a - adjust validation format for tests

- [Marcin Junczys-Dowmunt] a11df6a556b7f6f7fd4ca8017f9c333b0692783b - fix expected output for mnist

No changes

[View full output](http://vali.inf.ed.ac.uk/jenkins/job/marian-regression-tests-with-cudnn/100/)

**Comment by snukky** [comment]
I disabled those tests as the CUDNN staff is not expected to be working currently .
